the-collab-lab/tcl-38-smart-shopping-list
# Add tailwind styling to mobile home page

## Description

This PR adds the foundational styling for the app using Tailwind CSS. Styling added includes the overall body styling (background, typography, container height/width), primary button component, and mobile home page based on the Figma design. 

## Related Issue

Closes #43. 

## Acceptance Criteria

- Style the Homepage according to the design in Figma, including but not limited to:
- Setting the background image to [chalkboard image outlined in Figma]
- Adding the logo to the top of the layout
- Setting the fonts for each section [font A for headings, font B for items in the list and messages]
- Set the font color to [white]
- Set the container height/width parameters for mobile/desktop
- The join list / token form should have logo/flourishes per the design
- Styling the primary text buttons that will be used throughout the app with padding, margins, hover, and active states

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

| Before | After |
|---|---|
| ![image](https://user-images.githubusercontent.com/40015502/155247595-f1a2927d-20cf-4af3-8735-5686e295926e.png)| ![image](https://user-images.githubusercontent.com/40015502/155868823-68093f18-92f9-4992-9b16-f949b17c07f5.png) |


## Testing Steps / QA Criteria

Pull down the branch and compare styling to the [Figma](https://www.figma.com/file/dsTg5iRxSMbchiMjYT8mad/collab?node-id=0%3A1) mobile design for the home page. Please note that the desktop styling has not been added yet. Please ensure there are no changes to the form behavior.
